f8a_report/report_helper.py

```diff
@@ -363,7 +294,7 @@ def normalize_worker_data(self, start_date, end_date, stack_data, worker, freque
         """Normalize worker data for reporting."""
         # Adding some dummy comments because of low maintainability index.
         # This needs to be fixed by the original author.
-        total_stack_requests = {'all': 0, 'npm': 0, 'maven': 0}
+        total_stack_requests = {'all': 0, 'npm': 0, 'maven': 0, 'pypi': 0}
```

Comment:
is there no way to make the ecosystem thing generic? everytime we will have to add it like this otherwise

Reply:
same comment applies to all the other places where pypi was added

Reply:
@yzainee I am planning to do code improvements in the next PR. Keeping S3Helper in a separate file is a step towards that.